clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/HubConnectionBuilder.java

```diff
@@ -30,11 +37,15 @@ public HubConnectionBuilder configureLogging(Logger logger) {
         return this;
     }
 
-    public HubConnection build() throws Exception {
-        if (!built) {
-            built = true;
-            return new HubConnection(url, transport, logger);
+    public HubConnectionBuilder shouldSkipNegotiate(boolean skipNegotiate) {
```

Comment:
Remove

Reply:
Why

Reply:
None of the other clients have it. If we're going to add it, we should add it to all clients.

In general we strongly advise users not to use this setting because it means you can't be compatible with the Service. Setting this on your client would mean that if the server later chose to use the Service, they'd have to re-ship all their clients.